tests/dashboards/dashboard-management.spec.ts

```diff
@@ -110,73 +79,3 @@ test('as a user, I can delete a dashboard', async ({ page }) => {
     'Successfully deleted dashboard "My Dashboard".',
   );
 });
-
-test('as a user, I can delete multiple dashboards', async ({ page }) => {
```

Comment:
don't need to cover this use case?

Reply:
Yes, but perhaps not as a UI test. I'm thinking going beyond the basic workflows with UI tests is not working out. I'll prioritize backfilling the test.